--- setup.py	2014-07-02 06:07:42.000000000 -0400
+++ new.setup.py	2014-07-02 16:47:17.000000000 -0400
@@ -75,17 +75,6 @@
 if sys.version_info < (2, 7, 0):
     tests_require.append('unittest2')
 
-install_requires = ['PyYAML >=3.10', 'argh >=0.24.1', 'pathtools >=0.1.1']
-if sys.version_info < (2, 7, 0):
-    # argparse is merged into Python 2.7 in the Python 2x series
-    # and Python 3.2 in the Python 3x series.
-    install_requires.append('argparse >=1.1')
-    if any([key in sys.platform for key in ['bsd', 'darwin']]):
-        # Python 2.6 and below have the broken/non-existent kqueue implementations
-        # in the select module. This backported patch adds one from Python 2.7,
-        # which works.
-        install_requires.append('select_backport >=0.2')
-
 with open('README.rst') as f:
     readme = f.read()
 
@@ -135,7 +124,6 @@
       package_dir={'': SRC_DIR},
       packages=find_packages(SRC_DIR),
       include_package_data=True,
-      install_requires=install_requires,
       tests_require=tests_require,
       cmdclass={
           'build_ext': build_ext,
